### PR TITLE
Dark Theme

### PR DESCRIPTION
**Pull request by Ashish0804**
* Dark Theme
* More welcoming font
* Transparent icon 
* Different color for "Back to home page"
* Links open in New tabs

### CODE REVIEW

**Comment by tm9k1** [comment]
resolve the conflicts, @Ashish0804

**Comment by Ashish0804** [comment]
> resolve the conflicts, @Ashish0804

I am sorry i somehow missed it. Hope its all fine now @brute4s99

**Comment by Ashish0804** [comment]
Hey @brute4s99  i hope all the conflicts are resolved now.

**Comment by tm9k1** [comment]
@AbhishekChd @theLimitBreaker could you two review this please?

File: index.html [hunk 1]
```diff
@@ -11,14 +11,14 @@
...
-        <img class="header-logo" alt="OpenCode-MAIT logo image" src="assets/img/oc_logo.jpg">
+        <img class="header-logo" alt="OpenCode-MAIT logo image" src="assets/img/oc_logo.png">
         <h1> OpenCode-MAIT</h1>
         <p>Welcome to the Free and Open Source Software Community of MAIT.</p>
     </section>
 
     <section class='social-links'>
       <ul id='listtop'>
-        <li><a title='About Us' alt="About Us" href="about">About Us</a></li>
+        <li><a title='About Us' alt="About Us" href="about.html">About Us</a></li>
```
Comment (RotBolt) [inline index.html:21]: In the dark theme. `About Us , Github and Social` Links  backgrounds (only that portion) are white just like normal theme. You might want to change their background color in dark theme too.

Rest LGTM

**Comment by RotBolt** [comment]
@Ashish0804 for UI changes Please Upload screenshots for the change. It would be easier to review the changes .

**Comment by tm9k1** [comment]
@ashish0804 can you host your branch over gh-pages so I could test it?

**Comment by Ashish0804** [comment]
@brute4s99 this PR old. I'll close this PR and make a new one after working on the new repo and properly optimizing everything.

**Comment by tm9k1** [comment]
Looking forward to it! :D
All the best! \o/

**Pull request closed without merging**